lib/generators/templates/spec.erb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+RSpec.describe <%= class_name %> do
+  describe 'call' do
```

Comment:
I also followed http://betterspecs.org/ best practices here as well so this helps guide people to writing better specs then leaving them to write them however they see fit, which they can still do by deleting this code if they want.